From 6d8547b76c94622417613d97d9e3ba18ea012058 Mon Sep 17 00:00:00 2001
From: Zhuying Li <zhuying.li@amlogic.com>
Date: Wed, 26 Jan 2022 16:22:30 +0800
Subject: [PATCH] Bootanimation: T7 64 bit project bring up [1/1]

PD#SWPL-70914

Problem:
TvInputManagerService can't start due to lost media hal 64 bit so.

Solution:
Start service TvInputManagerService when tv is booting up.
media hal has provided 64 bit so.

Verify:
Verfiy on t7_an400

Signed-off-by: Zhuying <zhuying.li@amlogic.com>
Change-Id: Id70f2b47198c29b0027844caca17aebe57458631
---
 cmds/bootanimation/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmds/bootanimation/Android.bp b/cmds/bootanimation/Android.bp
index c3093a9b6530..b58932d8d7a6 100644
--- a/cmds/bootanimation/Android.bp
+++ b/cmds/bootanimation/Android.bp
@@ -9,6 +9,7 @@ cc_defaults {
         "-Werror",
         "-Wunused",
         "-Wunreachable-code",
+        "-Wno-format",
     ],
 
     shared_libs: [
-- 
2.25.1

